commit d9964c7aebb0034e28bd4e7b17f39ef5659cceac
Author: Louwrens van Dellen <lthvd@protonmail.com>
Date:   Wed May 29 17:04:02 2019 +0200

    ToFu shouldn't try git commands on a source extracted from tarball.

diff --git a/setup.py b/setup.py
index df6389d..c2d3b8d 100644
--- a/setup.py
+++ b/setup.py
@@ -70,19 +70,10 @@ if sys.version[:3] in ['2.7','3.6','3.7']:
 else:
     raise Exception("Pb. with python version in setup.py file: "+sys.version)
 
+git_branch = None
 if sys.version[0]=='2':
-    git_branch = subprocess.check_output(["git",
-                                          "rev-parse",
-                                          "--symbolic-full-name",
-                                          "--abbrev-ref",
-                                          "HEAD"]).rstrip()
     extralib = ['funcsigs']
 elif sys.version[0]=='3':
-    git_branch = subprocess.check_output(["git",
-                                          "rev-parse",
-                                          "--symbolic-full-name",
-                                          "--abbrev-ref",
-                                          "HEAD"]).rstrip().decode()
     extralib = []
 
 here = os.path.abspath(os.path.dirname(__file__))
